demo/admin/src/products/future/ProductForm.cometGen.ts

```diff
@@ -52,8 +52,9 @@ export const ProductForm: FormConfig<GQLProduct> = {
                     type: "asyncSelect",
                     name: "manufacturer",
                     rootQuery: "manufacturers",
-                    filterField: {
-                        name: "type",
+                    filter: {
+                        type: "prop",
+                        name: "manufacturerCountry",
                         gqlName: "addressAsEmbeddable_country",
```

Comment:
gqlName isn't used for the prop variant, right?

Reply:
no, it's used. this is required to detect the correct gql-filterArg-field (required for correct prop-type) if the prop-name should be different than how the gql-filterArg-field is called.

see the example: prop is named `manufacturerCountry` but the name of the gql-filterArg-field is called  `addressAsEmbeddable_country`. Else it would be required to have a prop named `addressAsEmbeddable_country`. nobody would ever name a prop like this. In some cases, e.g. without nested filter, name will be probably enough.